Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ghcr.io/rse-ops/cuda-ubuntu-20.04:cuda-11.0.3
+
+# docker build -t ghcr.io/mfem/mfem-ubuntu .
+
+RUN apt-get update && \
+    apt-get install -y unzip gfortran && \ 
+    spack compiler find && \
+    spack install sw4lite && \
+    apt-get install -y libcurl4-openssl-dev libssl-dev && \
+    spack install liblas lapackpp && \
+    git clone --depth 1 https://github.com/geodynamics/sw4 /code
```

Comment:
This looks like an artifact. sw4 is not a dependency or the like.

Reply:
Yep an oversight (and I will remove) - I started from another Dockerfile I was working on.

Let me know about the things I proposed above and when I am allowed to push changes.

Reply:
Gotcha! I'll push changes (without force) as they are requested.

Reply:
okay that makes it much faster (53 minutes!) I'm still confused why the action doesn't end at the end - it's either cleaning something up or a legit GitHub bug. I can open a ticket if it doesn't seem to ever end, I've never seen this before.

Reply:
♾️🐳

Reply:
@jandrej your emoji game is on point :point_right: :pencil2: